### PR TITLE
Updated workspace

### PR DESCRIPTION
**Pull request by FWachter**
YuMi was put into a new workspace. Modified relevant files for new workspace.

### CODE REVIEW

File: yumi_description/urdf/workspace.xacro [hunk 1]
```diff
@@ -17,28 +17,51 @@
     <joint name="world_to_table" type="fixed">
       <parent link="${table_parent}"/>
       <child link="table"/>
-      <insert_block name="origin"/>
+      <origin rpy="0 0 0" xyz="0.334 -0.01 -0.013"/>
```
Comment (ntonci) [inline yumi_description/urdf/workspace.xacro]: These values should be parameters

Comment (FWachter) [inline yumi_description/urdf/workspace.xacro]: As in I should revert back to the way it was before?

**Comment by FWachter** [comment]
Updated branch according to Tonci's comment. Any other modifications or am I good to merge?

File: yumi_moveit_config/config/yumi.srdf [hunk 2]
```diff
@@ -285,18 +285,21 @@
...
-    <disable_collisions link1="camera_assembly" link2="creative_camera" reason="Never" /> <!-- Camera Assembly and Creative Camera -->
-    <disable_collisions link1="camera_assembly" link2="vi_sensor" reason="Never" /> <!-- Camera Assembly and VI Sensor -->
-    <disable_collisions link1="camera_holder" link2="creative_camera" reason="Adjacent" /> <!-- Camera Holder and Creative Camera -->
-    <disable_collisions link1="camera_holder" link2="vi_sensor" reason="Adjacent" /> <!-- Camera Holder and VI Sensor -->
-    <disable_collisions link1="creative_camera" link2="vi_sensor" reason="Never" /> <!-- Creative Camera and VI Sensor -->
+    <disable_collisions link1="yumi_link_7_l" link2="camera_assembly" reason="Adjacent"/> <!-- Link 7 Left Hand to Camera Assembly -->
+    <disable_collisions link1="yumi_link_7_l" link2="camera_holder" reason="Adjacent"/> <!-- Link 7 Left Hand and Camera Holder -->
+    <disable_collisions link1="camera_assembly" link2="camera_holder" reason="Adjacent"/> <!-- Camera Assembly and Camera Holder -->
+    <disable_collisions link1="camera_assembly" link2="creative_camera" reason="Never"/> <!-- Camera Assembly and Creative Camera -->
```
Comment (ntonci) [inline yumi_moveit_config/config/yumi.srdf]: Can you rename creative_camera to sr300_sensor

File: yumi_moveit_config/config/yumi.srdf [hunk 3]
```diff
@@ -285,18 +285,21 @@
...
+    <disable_collisions link1="camera_assembly" link2="vi_sensor" reason="Never"/> <!-- Camera Assembly and VI Sensor -->
+    <disable_collisions link1="camera_holder" link2="creative_camera" reason="Adjacent"/> <!-- Camera Holder and Creative Camera -->
```
Comment (ntonci) [inline yumi_moveit_config/config/yumi.srdf]: Same here.
Other than that lgtm.

**Pull request merged**